agent/app/agent_windows.go

```diff
@@ -279,5 +279,9 @@ func (agent *ecsAgent) getPlatformDevices() []*ecs.PlatformDevice {
 }
 
 func (agent *ecsAgent) loadPauseContainer() error {
-	return nil
+	//The pause image would be cached in th ECS-Optimized Windows AMI's and will be available. We will throw an error if the image is not loaded.
+	//If the agent is run on non-supported instances then pause image has to be loaded manually by the client.
+	_, err := agent.pauseLoader.IsLoaded(agent.dockerClient)
```

Comment:
> We will throw an error if the image is not loaded

then shouldn't you check the bool returned here and see if its true?

Reply:
If the image failed to load then the err will not be nil. We will check the same in agent.go (https://github.com/aws/amazon-ecs-agent/blob/3776bee911c2267dedce46a9d2a17007f0bd73ae/agent/app/agent.go#L261)

The usage of bool is redundant in our case.